### DISCUSSION TITLE
Customizing shimmering focus style settings for use on publish

### DISCUSSION THREAD

**Discussion by AlexRimor** [chrisgrieser/shimmering-focus#167] [Q&A] [answered]
This theme has excellent features that make it great to work with locally. 

This is because we can use the Style settings plugin to choose options and tweak things. 
However, none of these options work when the theme is used on Publish. 

There are other variables as well, which might be handled by the core application such as the text size. On my computer I can set the font size to 13px. Rendered on the web - it renders as 18px which is too big.  
 
Is there a way to toggle the settings you've already built for use in desktop to be used in Publish?
I can go in there and manually change them if need be. 

Also posted on the [obsidian forum here](https://forum.obsidian.md/t/how-do-i-use-theme-options-from-the-style-settings-plugin-with-obsidian-publish/37302).

**Comment by chrisgrieser** [comment, marked as answer]
Hi! Unfortunately, this theme doesn't have any support for Publish. And since I do not use publish myself, it's rather unlikely that publish support will be implemented in the foreseeable future, sorry!
